anesthetic/plot.py

```diff
@@ -292,11 +292,6 @@ def make_2d_axes(params, **kwargs):
                     axes[x][y].twin = axes[x][y].twinx()
                     axes[x][y].twin.set_yticks([])
                     axes[x][y].twin.set_ylim(0, 1.1)
-                    axes[x][y].set_zorder(axes[x][y].twin.get_zorder() + 1)
```

Comment:
This line had me surprised that our code actually worked in the past. Leaving this in will render the diagonal plots invisible now...

Is there a reason this line was/is important that I'm overlooking?

Reply:
This was introduced for version 1.1, and by my reckoning the functionality implemented here was superseded by the `make_diagonal` function. If the new API breaks this, and it seems to work without, let's tentatively drop.